nodejs/week1/warmup.js

```diff
@@ -0,0 +1,32 @@
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+
+    getDiameter() {
+        this.diameter = 2 * this.radius;
+        console.log(`The diameter is ${this.diameter}`);
+    }
+
+    getCircumference() {
+
+        this.circumference = 2 * Math.PI * this.radius;
+        console.log(`The circumference is ${this.circumference.toFixed(1)}`);
+    }
+
+    getArea() {
+        this.area = Math.PI * Math.pow(this.radius, 2);
+        console.log(`The area is ${this.area.toFixed(1)}`);
+
+    }
+}
+
+const circle = new Circle(2);
+circle.getDiameter();
```

Comment:
use a return value here, and do the console log here. Same for all the other functions